packages/flutter_test/lib/src/accessibility.dart

```diff
@@ -200,7 +200,7 @@ class MinimumTextContrastGuideline extends AccessibilityGuideline {
     final ByteData byteData = await tester.binding.runAsync<ByteData>(() async {
       // Needs to be the same pixel ratio otherwise our dimensions won't match the
       // last transform layer.
-      image = await layer.toImage(renderView.paintBounds, pixelRatio: 1.0);
+      image = await layer.toImage(renderView.paintBounds, pixelRatio: 1 / 3);
```

Comment:
Just curious, why the `1/3` change?  It seems like the comment above this line no longer fully holds.

Reply:
When I used the semantics node, we ended up including the dpr transform. Now that I'm using the rect from the render box, this doesn't include the dpr transform